pkg/log/zap/zap.go

```diff
@@ -207,3 +208,39 @@ func NewRaw(opts ...Opts) *zap.Logger {
 	log = log.WithOptions(o.ZapOpts...)
 	return log
 }
+
+// BindToFlagSet func allows CLI passed flags to be parsed.
+func (o *Options) BindFlags(fs *pflag.FlagSet) {
```

Comment:
since there's not short flags here, can we use use "flag" instead?  That was we're more compatible with non-pflag setups.